apps/settings/js/modules/panel_cache.js

```diff
@@ -32,6 +32,7 @@ define(function(require) {
                        'style/simcard.css',
                        'style/updates.css',
                        'style/downloads.css',
+                       'style/improve_browser_os.css',
```

Comment:
Since we are suffer from loadtime performance, I'd rather put single style sheet into settings.css